muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/sass/elements/application-list.scss

```diff
@@ -24,6 +25,135 @@
   }
 }
 
+.application-list__header {
```

Comment:
application-list__header-title perhaps as used in application-panel__header-title to keep consistency?

Reply:
Look above the above's above